src/apps/handlers/prepareMovementData.js

```diff
@@ -1,10 +1,21 @@
 export default function getMovementData(data) {
```

Comment:
`data` here is an `actor` so we should use that name to be more explicit. Would appreciate it if you could make that change along with the other stuff 🙂